src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final String randomColor = new ColorSupplier().getRandomColor();
+    private final int randomFigure = random.nextInt(5);
+
+    public Figure getRandomFigure() {
+        switch (randomFigure) {
```

Comment:
Make it return random figure for every method call.

Reply:
Still exists.